tycho-surefire/pom.xml

```diff
@@ -14,9 +14,8 @@
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>org.eclipse.tycho</groupId>
-		<artifactId>tycho-bundles</artifactId>
```

Comment:
Could this be the issue? I don't see why this change is in this PR and tycho-bundles parent pom overrides target-platform-configuration.

Reply:
Actually tycho-bundles as a parent is not required here as all submodules using non tycho-build (jar packaging) or directly use tycho-bundles as their parent....

Beside that I created a PR without any (code) changes and it also fails with this test :-\ 

See
- https://github.com/eclipse-tycho/tycho/pull/1547
- https://ci.eclipse.org/tycho/job/tycho-github/job/PR-1547/1/

so it is a bit strange ...